freqtrade/rpc/telegram.py

```diff
@@ -840,14 +841,20 @@ def _balance(self, update: Update, context: CallbackContext) -> None:
                             f"*{curr['currency']}:*\n"
                             f"\t`Available: {curr['free']:.8f}`\n"
                             f"\t`Balance: {curr['balance']:.8f}`\n"
-                            f"\t`Pending: {curr['used']:.8f}`\n"
-                            f"\t`Est. {curr['stake']}: "
-                            f"{round_coin_value(curr['est_stake'], curr['stake'], False)}`\n")
+                            f"\t`Pending: {curr['used']:.8f}`\n")
+                        if curr['stake'] == curr['stake']:
+                            reserved = self._rpc._freqtrade.wallets.get_reserved_stake_amount()
```

Comment:
This should probably be moved to `rpc.py` - and added to the stake-currency entry (instead of calling this here).

That way, it'll also be available for the API.